lib/OpenQA/Worker.pm

```diff
@@ -814,8 +814,7 @@ sub _clean_pool_directory ($self) {
         next if $excludes{$file};
         if (-d $file) {
             remove_tree($file);
-        }
-        else {
+        } else {
```

Comment:
?

Reply:
I was wondering what that function did, I think I went into autopilot and changed that ugly else under the parenthesis. Do you want me to revert the change?

Reply:
Perl tidy don't like this, so I'm reverting!